src/lib/styleHelper.js

```diff
@@ -0,0 +1 @@
+export const prefixer = prefix => className => `ncms-${ prefix }-${ className }`;
```

Comment:
We should document the prefixer. 

Is there a runtime impact of this implementation? Is my browser going to be building these strings every time it renders a component? If we're writing the style name in the css file, why not use a static name in the templates?